packages/web-components/src/components/number-input/number-input.mdx

```diff
@@ -1,78 +1,45 @@
-import { ArgTypes, Meta, Markdown } from '@storybook/blocks';
-import { cdnJs, cdnCss } from '../../globals/internal/storybook-cdn';
+import { ArgTypes, Canvas, Markdown, Meta } from '@storybook/blocks';
+import { cdnJs } from '../../globals/internal/storybook-cdn';
 import * as NumberInputStories from './number-input.stories';
 
 <Meta of={NumberInputStories} />
 
-# Number input
+# NumberInput
```

Comment:
hmm should the be kept as `Number input`. We have `File uploader` and `Content switcher` but we also have `DefinitionTooltip`